### PR TITLE
[SCB-2881] support setting hibernate validation configuration

### PR DESCRIPTION
**Pull request by chengyouling**

### CODE REVIEW

File: core/src/main/java/org/apache/servicecomb/core/filter/impl/ParameterValidatorFilter.java [hunk 1]
```diff
@@ -111,9 +113,14 @@ protected ValidatorFactory createValidatorFactory() {
         .configure()
         .propertyNodeNameProvider(new JacksonPropertyNodeNameProvider())
         .messageInterpolator(messageInterpolator())
+        .addProperty(HibernateValidatorConfiguration.FAIL_FAST, buildHibernateFailFastProperty())
         .buildValidatorFactory();
   }
 
+  private String buildHibernateFailFastProperty() {
+    return LegacyPropertyFactory.getStringProperty(HibernateValidatorConfiguration.FAIL_FAST, "false");
+  }
```
Comment (liubao68) [inline core/src/main/java/org/apache/servicecomb/core/filter/impl/ParameterValidatorFilter.java:122]: Using Environment, avoid using LegacyPropertyFactory if possible

Comment (chengyouling) [reply core/src/main/java/org/apache/servicecomb/core/filter/impl/ParameterValidatorFilter.java]: fixed

File: swagger/swagger-invocation/invocation-validator/src/main/java/org/apache/servicecomb/swagger/invocation/validator/ParameterValidator.java [hunk 2]
```diff
@@ -16,8 +16,10 @@
  */
 package org.apache.servicecomb.swagger.invocation.validator;
 
+import java.util.Map;
 import java.util.Set;
 
+import org.apache.servicecomb.config.ConfigUtil;
```
Comment (liubao68) [inline swagger/swagger-invocation/invocation-validator/src/main/java/org/apache/servicecomb/swagger/invocation/validator/ParameterValidator.java:22]: delete used ParameterValidator and ProducerInvokeExtension

Comment (chengyouling) [reply swagger/swagger-invocation/invocation-validator/src/main/java/org/apache/servicecomb/swagger/invocation/validator/ParameterValidator.java]: delete

File: core/src/main/java/org/apache/servicecomb/core/filter/impl/ParameterValidatorFilter.java [hunk 3]
```diff
@@ -17,9 +17,11 @@
 package org.apache.servicecomb.core.filter.impl;
```
Comment (liubao68) [inline core/src/main/java/org/apache/servicecomb/core/filter/impl/ParameterValidatorFilter.java:17]: delete ProducerInvokeExtension too

**Review by liubao68 — APPROVED** [review]

**Pull request merged**